src/app/common/form-utils.ts

```diff
@@ -9,6 +9,10 @@ export function useShowFieldError(name: string) {
   const form = useFormikContext();
   const [_, meta] = useField(name);
   const isDirty = useIsFieldDirty(name);
```

Comment:
This change means we won't show an error until the user un-focuses from the field. Validation will still run `onBlur` but the error won't be shown until they un-focus:

https://github.com/hirosystems/wallet/assets/2938440/8565e810-bc06-498c-9b99-56674c1e3d11

Without this fix, we show an error immediately as they are focused and entering any value, correct or not:


https://github.com/hirosystems/wallet/assets/2938440/3c2c57f0-723f-4e6f-bfae-47de7ddf8722